go/test/fuzzing/tablet_manager_fuzzer.go

```diff
@@ -22,7 +22,7 @@ import (
 	"vitess.io/vitess/go/mysql/fakesqldb"
 	"vitess.io/vitess/go/sqltypes"
 	"vitess.io/vitess/go/vt/dbconfigs"
-	"vitess.io/vitess/go/vt/mysqlctl/fakemysqldaemon"
+	"vitess.io/vitess/go/vt/mysqlctl"
```

Comment:
Moved `FakeMysqlDaemon` into `mysql` package so that I can use it in non-blackbox tests of `Backup` and `Restore`.